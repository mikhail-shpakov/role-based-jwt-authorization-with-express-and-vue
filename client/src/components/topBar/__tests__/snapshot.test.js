import { createLocalVue, shallowMount } from '@vue/test-utils'
import Buefy from 'buefy'
import topBar from '../index'

describe('./index.vue', () => {
  const localVue = createLocalVue()
  localVue.use(Buefy)

  it('when load page, topBar renders correctly', () => {
    const wrapper = shallowMount(topBar, {
      localVue
    })
    expect(wrapper).toMatchSnapshot()
  })
})
