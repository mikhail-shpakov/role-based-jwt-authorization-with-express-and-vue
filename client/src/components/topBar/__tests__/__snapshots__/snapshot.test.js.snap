// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`./index.vue when load page, topBar renders correctly 1`] = `
<div class="section">
  <nav role="navigation" aria-label="main navigation" class="navbar navbar-custom">
    <div class="navbar-brand"><a href="https://timeweb.com/" target="_blank" rel="noopener" class="navbar-item item-logo"><img src="@/image/logo.svg" alt="img-logo-timeweb"></a></div>
    <div class="navbar-menu">
      <div class="navbar-start"></div>
      <div class="navbar-end"></div>
    </div>
  </nav>
</div>
`;
